dev/io.openliberty.pages.3.1.internal_fat/fat/src/io/openliberty/pages31/fat/tests/TagsTestsEE10.java

```diff
@@ -0,0 +1,189 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.pages31.fat.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.logging.Logger;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import io.openliberty.pages31.fat.JSPUtils;
+import com.meterware.httpunit.GetMethodWebRequest;
+import com.meterware.httpunit.WebConversation;
+import com.meterware.httpunit.WebRequest;
+import com.meterware.httpunit.WebResponse;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+
+// For testFormatTags
+import java.util.Date;
+import java.text.DateFormat;
+import java.util.Locale;
+
+/**
+ *  Smoke Tests for JSTL EE10 Tags
```

Comment:
Are these just the EE9 tests in the EE10 bucket?

https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSTLTests.java

If we repeat the JSP 2.3 bucket for EE10 why are these necessary? Are they different in some way?

Reply:
This EE10 class test the new jakarta.tags.* URIs.  The EE9 one was for the older sun URIs. 

I think the older URIs are also tested again EE10 in the com.ibm.ws.jsp2.3_fat (JSTLTests.java) because we kept backwards compatibility.

Reply:
But otherwise, the tests are exactly the same.

Reply:
Can you add a comment with that information so when someone looks at this test class it is obvious to them?